package.json

```diff
@@ -1,11 +1,25 @@
 {
   "private": true,
   "version": "2.9.0",
-  "engines": {
-    "node": "^10.24.1"
-  },
+  "browserslist": [
+    "maintained node versions",
+    "supports es6-module and > 0.5%",
+    "not dead",
+    "not IE 11",
```

Comment:
I think everything below "not IE11" is unnecessary. May be just ignore safari explicitly. You can see which browser versions are targeted by adding `"debug": true` to `babel-preset-env` options.